cumulus/parachains/runtimes/bridge-hubs/bridge-hub-westend/src/bridge_to_ethereum_config.rs

```diff
@@ -139,32 +139,40 @@ parameter_types! {
 		deneb: Fork {
 			version: [4, 0, 0, 0], // 0x04000000
 			epoch: 0,
-		}
+		},
+		electra: Fork {
+			version: [5, 0, 0, 0], // 0x05000000
+			epoch: 2000000,
+		},
 	};
 }
 
 #[cfg(not(any(feature = "std", feature = "fast-runtime", feature = "runtime-benchmarks", test)))]
 parameter_types! {
 	pub const ChainForkVersions: ForkVersions = ForkVersions {
 		genesis: Fork {
-			version: [144, 0, 0, 111], // 0x90000069
+			version: [16,99,118,36], // 0x10637624
 			epoch: 0,
 		},
 		altair: Fork {
-			version: [144, 0, 0, 112], // 0x90000070
-			epoch: 50,
+			version: [32,99,118,36], // 0x20637624
+			epoch: 0,
 		},
 		bellatrix: Fork {
-			version: [144, 0, 0, 113], // 0x90000071
-			epoch: 100,
+			version: [48,99,118,36], // 0x30637624
+			epoch: 0,
 		},
 		capella: Fork {
-			version: [144, 0, 0, 114], // 0x90000072
-			epoch: 56832,
+			version: [64,99,118,36], // 0x40637624
+			epoch: 0,
 		},
 		deneb: Fork {
-			version: [144, 0, 0, 115], // 0x90000073
-			epoch: 132608,
+			version: [80,99,118,36], // 0x50637624
+			epoch: 0,
+		},
+		electra: Fork {
+			version: [96,99,118,36], // 0x60637624
+			epoch: 256,
 		},
```

Comment:
Just curious why changing the `ForkVersions` here, IIUC this is the configuration for the `Westend-Sepolia` as declared in https://github.com/eth-clients/sepolia/blob/da5654742513435bdd6dbc5fd033cf593ce57a0f/metadata/config.yaml#L36

Will that break the testnet?

Reply:
Ah good catch. :) I changed the epoch to a placeholder and added a comment in https://github.com/paritytech/polkadot-sdk/pull/7075/commits/0b691062ff4307ed67221a26b117324c3bdbda9d.

The fork version `0x60637624` has not been announced yet, but since that seems to be the format of the Sepolia fork versions, that is likely the fork version they'll use. For now, it does not really matter since the Electra epoch will not be reached yet, since a placeholder epoch is set.

Reply:
@yrong I realized now `0x60637624` is actually from the Mekong testnet, which I used to test the upgrade. 😅 Switched back to Sepolia in https://github.com/paritytech/polkadot-sdk/pull/7075/commits/f758355a3a808078f97ac88b00fa26bccbccbcff